artifactory.py

```diff
@@ -1724,6 +1743,32 @@ def deploy_file(
                 explode_archive_atomic=explode_archive_atomic,
             )
 
+    def deploy_by_checksum(
+        self,
+        sha1=None,
+        sha256=None,
+        checksum=None,
+        parameters={},
+    ):
+        """
+        Deploy an artifact to the specified destination by checking if the
+        artifact content already exists in Artifactory.
+
+        :param pathobj: ArtifactoryPath object
+        :param sha1: sha1Value
```

Comment:
```suggestion
        :param sha1_val: sha1Value
```

otherwise might confuse compared to deploy_file where sha1 is a bool

Reply:
It seems the bool parameter is `cacl_sha1` in `deploy_file`. 
And `sha1` always indicates the SHA1 digest of the file

Reply:
@zhan9san 
you are right